src/services/README.md

```diff
@@ -0,0 +1,44 @@
+# Services
+Homeday Blocks contains a number of services that we found ourself needing across projects. Some of them are used by Homeday Blocks components, others 
+are here to be consumed in projects directly.
+
+Index:
+- [Breakpoints service](https://github.com/homeday-de/homeday-blocks/tree/develop/src/services#breakpoints) - Provides a wrapper component for conditional rendering based on viewport size
+
+## Breakpoints
+Exposes a wrapper component that helps with conditional rendering based on viewport size. Relies on the [onResize](#on-resize) service.
+
+### Usage
+In your project entry file, import the service and define breakpoints that make sense for your project:
+
+```js
+import { setBreakpoints } from 'hd-blocks/services/breakpoints';
+
+const breakpoints = {
+  s: '(max-width:599px)',
+  m: '(min-width:600px)',
+  l: '(min-width:900px)',
+  xl: '(min-width:1200px)',
+};
+setBreakpoints(breakpoints);
+```
+
+You can now import a helper component and render contents based on your breakpoints matching or not:
+
+```vue
+<script>
+import { Responsive } from 'hd-blocks/services/breakpoints';
+</script>
+
+<template>
+<Responsive #default="{ matches, indeterminate }">
```

Comment:
Can you please add example of `matches` and `indeterminate`? I think is not fully clear how those object look like, even though, there is a description under.

Reply:
Will do :)

Reply:
I have added the examples into the docs immediately below.